src/components/Image.astro

```diff
@@ -0,0 +1,14 @@
+---
+// Define the component's properties
+type Props = {
+  imgSrc: string;
```

Comment:
For the props, do we need to prefix with `img` as the component is called `Image`.

i.e. `<Image src="" alt="" caption="" />` makes sense instead of `<Image imgSrc="" imgAlt="" caption=""" />`